internal/ui/ui_glfw.go

```diff
@@ -265,6 +320,75 @@ func (u *userInterfaceImpl) setRunning(running bool) {
 	}
 }
 
+func (u *userInterfaceImpl) setWindowMonitor(monitor int) {
+	if microsoftgdk.IsXbox() {
+		return
+	}
+
+	u.m.RLock()
+	defer u.m.RUnlock()
+
+	if monitor >= 0 && len(monitors) > monitor {
```

Comment:
I think we can panic when `monitor < 0 || len(monitors) <= monitor`. This can never happen basically.

Reply:
Hmm... I think this call is entirely unnecessary... let me check out codes paths.

Reply:
I have removed that and another unnecessary range check that can never happen. I don't think it is worthwhile to panic, since it can't happen. If you'd prefer to panic, we could still.